examples/tracer-web/webpack.dev.config.js

```diff
@@ -1,4 +1,6 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
If I recall correctly, we can get this rule to work as expected (not reporting on toolings) by adding a "files" field in the relevant package.json to exclude these tooling files.

Reply:
Adding files didn't work, so I'm adding a devDependencies exemption to the examples .eslintrc file